web/src/components/common/ListingCollection.tsx

```diff
@@ -44,44 +44,41 @@ const ListingItem: React.FC<ListingProps> = ({
     minCommits,
     imgUrl,
   },
-}) => (
-  <StrippedCol xs={6} key={id}>
-    <Link
-      to={{
-        pathname: `/listing/${id}`,
-        state: {
-          fromExplore: true,
-        },
-      }}
-    >
-      <ListingCard
-        listingName={name}
-        price={price}
-        oldPrice={oldPrice}
-        endDate={deadline}
-        imgUrl={imgUrl}
+}) => {
+  const isMobile = useMediaQuery({query: '(max-device-width: 768px)'});
```

Comment:
Would prefer values like `768px` to be in a new `constants/media-query.ts` file instead! It can probably be called `SM_MIN`? 

Here is the bootstrap grid system, and we can probably use their sizes as a guide for our constants file :D  
```
xs (for phones - screens <768px wide)
sm (for tablets - screens =>768px wide)
md (for small laptops - screens =>992px wide)
lg (for laptops and desktops - screens =>1200px wide)
```
These constants can then be used in `useMediaQuery`s and also in `@media` in our css styling.

Reply:
@hzxcaryn Okay! Will do something like that.